utils/include/utils/string_operations.h

```diff
@@ -21,7 +21,44 @@
 
 namespace hlasm_plugin::utils {
 
-std::pair<std::string_view, size_t> trim_left(std::string_view s);
+template<typename S>
+size_t trim_left(S& s)
+{
+    if (s.empty() || s.front() != ' ')
+        return 0;
+
+    const auto to_trim = s.find_first_not_of(' ');
+    if (to_trim == S::npos)
+    {
+        auto s_length = s.length();
+        s = {};
+        return s_length;
+    }
+
+    s.remove_prefix(to_trim);
```

Comment:
`S` can really only be std::string_view...

Reply:
Template removed and reworked related code in `low_language_processor.cpp`